src/highrl/envs/robot_env.py

```diff
@@ -314,46 +240,46 @@ def render(
 
         # Create viewer
         if self.viewer is None:
-            self.viewer = rendering.Viewer(self.width, self.height)
+            self.viewer = rendering.Viewer(self.cfg.width, self.cfg.height)
             self.transform = rendering.Transform()
             self.transform.set_scale(10, 10)
             self.transform.set_translation(128, 128)
             self.score_label = pyglet.text.Label(
                 "0000",
                 font_size=12,
                 x=20,
-                y=int(self.height * 2.5 / 40.00),
+                y=int(self.cfg.height * 2.5 / 40.00),
                 anchor_x="left",
                 anchor_y="center",
-                color=white_color,
+                color=colors.white_color,
             )
             self.iteration_label = pyglet.text.Label(
                 "0000",
                 font_size=12,
                 x=20,
-                y=int((self.height * 1.6) // 40.00),
+                y=int((self.cfg.height * 1.6) // 40.00),
                 anchor_x="left",
                 anchor_y="center",
-                color=white_color,
+                color=colors.white_color,
             )
             self.transform = rendering.Transform()
             self.image_lock = threading.Lock()
 
-        def make_circle(c, r, res=10):
+        def make_circle(coords: Tuple[int, int], radius: int, res=10) -> np.ndarray:
             """Create circle points
 
             Args:
-                c (list): center of the circle
-                r (float): radius of the circle
+                coords (list): center of the circle
+                radius (float): radius of the circle
```

Comment:
`radius` var should be of type int not float

Reply:
Done

Reply:
Done